packages/arb-token-bridge-ui/scripts/generateCoreChainsToMonitor.ts

```diff
@@ -34,6 +34,12 @@ async function generateCoreChainsToMonitor() {
     })
   )
 
+  // Ensure the public directory exists
+  const publicDir = './public'
+  if (!fs.existsSync(publicDir)) {
+    fs.mkdirSync(publicDir, { recursive: true })
+  }
```

Comment:
We reuse `./public` on L51, maybe we can update it there too.

This scripts is running from `arb-token-bridge-ui`, and `public` folder is now in `packages/app` folder. I would reuse this public folder rather than creating one here